README.md

```diff
@@ -1,193 +1,48 @@
-
-Installation will take a while, due to some large downloads, and long installation steps. Please be patient.
-
-You might want to take a [snapshot](https://www.virtualbox.org/manual/ch01.html#snapshots) of the VMs at this stage.
-
-### Teardown
-
-To uninstall, use the 'uninstall' subcommand.
-```bash
-./romana-setup -p vagrant uninstall
-```
-
-## Using The System
+* [core](https://github.com/romana/core ): A number of micro services written in Go, which comprise the core components of the Romana system.
+* [kube](https://github.com/romana/kube): The Romana CNI plugin and Network Policy Agents for Kubernetes
+* [networking-romana](https://github.com/romana/networking-romana): The Romana ML2 plugin and IPAM driver for OpenStack
 
-At the end of installation, you should see a summary containing the URL of the dashboard,
-and SSH commands to use to connect to the instances.
+The READMEs of those repos contain more information about the source code and how to run and test it.
 
-Other things you may wish to do:
-- launch an instance using Horizon:
-  * Log into the dashboard using username `admin` and password `secrete` (or the password you configured before installation)
-  * Select `Instances` from the `Project/Compute` sidebar
-  * Click the `Launch Instance` button
-  * Provide the required details
-  * Optionally, select the Advanced tab and specify a segment name in the `Romana Network Segment` field
-  * Click the Launch button
-- launch an instance using command-line: `nova boot --flavor m1.nano --image cirros-0.3.4-x86_64-uec --nic net-id=$(neutron net-show romana -Fid -f value) --meta romanaSegment=default instance-name`
-- connect to the instance: `ssh cirros@instance-ip`
-- install an ubuntu image:
-  * Create a new flavor with RAM:512MB, Disk: 3GB, VCPUs: 1: `nova flavor-create m1.smallish auto 512 3 1`
-  * Download a suitable image: `wget https://cloud-images.ubuntu.com/releases/14.04.3/release/ubuntu-14.04-server-cloudimg-amd64-disk1.img`
-  * Create image: `glance image-create --visibility public --disk-format qcow2 --container-format bare --name "ubuntu" < ubuntu-14.04-server-cloudimg-amd64-disk1.img`
-- register a new host: on the controller, run `romana add-host <hostname> <host-ip> <romana-cidr> <agent-port>`
-- add a new tenant: on the controller, run `romana create-tenant <tenant-name>`
-- add a new segment: on the controller, run `romana add-segment <tenant-name> <segment-name>`
-- see a list of tenants: `romana show-tenant`, details for a specific tenant: `romana show-tenant <tenant-name>`
-- see a list of hosts: `romana show-host`, details for a specific host: `romana show-host <hostname>`
+#  Installation
 
-See also: [Try Romana Now](http://romana.io/try_romana/openstack/)
+To get up and running with Romana, some scripts and Ansible playbooks have been provided to automate the setup and deployment.
+It currently supports [Amazon EC2](https://aws.amazon.com/ec2/) instances and [Vagrant](https://www.vagrantup.com/) VMs,
```

Comment:
I would re-phrase this a little, since "integrated" seem to imply a little bit too much.

Maybe say something like:

"This installer currently is capable of setting up a stand-alone Kubernetes or OpenStack-Devstack cluster. As deployment targets for those clusters, it supports Amacon EC2 or local Vagrant VMs."